app/javascript/css/tailwind.scss

```diff
@@ -61,6 +61,9 @@
  * @import "utilities/skew-transforms";
  */
 
+.whitetext {
```

Comment:
Why are we having this class?
You can use the tailwind `text-white`

Reply:
We are not using this class, removed